catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -55,6 +57,10 @@
 
     private static final String TEMP_FILE_EXTENSION = ".tmp";
 
+    private static final ExecutorService EXECUTOR_SERVICE =
+            Executors.newFixedThreadPool(Runtime.getRuntime()
```

Comment:
Can we just use a single thread executor here? That should help the ingest performance a little more.